### ISSUE TITLE
semantic-ui not working from within fullpage-react

### ISSUE THREAD

**Issue by foobar8675** [cmswalker/fullpage-react#32]
Hi @cmswalker,

First, thanks for writing this. It is really helpful. I'm using http://react.semantic-ui.com/ and many of the semantic-ui features are not working when called from within fullpage-react. I was wondering if this or something like this is something you might have come across?

**Comment by cmswalker** [comment]
@foobar8675 Thanks for reporting.

So i haven't looked into this a ton, but I downloaded react-semantic-ui and tried to use it and you're right there are conflicts. What is odd is that the conflicts only occur when I load in their css (which require jQuery) and I'm afraid that is setting up event listener and scroll conflicts. https://github.com/Semantic-Org/Semantic-UI-CSS/search?utf8=%E2%9C%93&q=scroll

I'll try to look into this but I'm not sure if it can be resolved, the 2 may just not play nicely together. I had to make some decisions with fullpage-react to overtake scroll events in order to provide an easy API and I'm not sure I can bend it for semantic-ui.  I'd suggest trying a more lightweight UI framework.
